src/components/app.js

```diff
@@ -74,9 +40,24 @@ class App extends React.Component {
             </div>
           </div>
           <div className='content-block'>
-            <div className='content-container'>
-              <Route exact path='/' component={Introduction} />
-              <Route path='/:componentKey' render={getComponent} />
+            <div className='content-container ds-l-row'>
+              <div className='ds-u-float--left ds-u-padding-right--6 ds-u-padding-top--2 ds-l-col--3'>
+                <ul className='ds-c-vertical-nav__subnav'>
+                  {LeftNav}
+                </ul>
+              </div>
+              <div className='ds-u-float--left ds-u-padding--1 ds-l-col--9'>
+                <div className='ds-u-measure--wide'>
+                  {mergedRoutes.map((route, index) => (
```

Comment:
do you think destructuring path, exact, and component from route here would be more clear?